src/executer/executer.c

```diff
@@ -71,9 +71,15 @@ void	pipe_and_fork(t_command *current, t_shell *shell)
 void	execute_command(t_command *current, t_shell *shell)
 {
 	signal(SIGQUIT, SIG_DFL);
-	if (!current->cmd_name[0])
-	{
-		shell->exit_status = 0;
+	// if (!current->cmd_name[0])
+	// {
+	// 	shell->exit_status = 0;
+	// 	free_and_exit_shell(shell, shell->exit_status);
+	// }
+	if (!current->cmd_name || !current->cmd_name[0] || current->cmd_name[0][0] == '\0')
```

Comment:
This conditions causes errors for echo $"", echo "" $HOME and others, it returns command not found instead of printing \n and returning the prompt. Needs to be slightly modified.